css/templates/e-commerce/header.css

```diff
@@ -0,0 +1,126 @@
+@import '../../core';
+@import 'page-vars';
+
+.ampstart-icon {
+  fill: var(--color-primary);
```

Comment:
Moved to color-icon, defined a base value as color-accent and overridden in ecommerce template as color-primary